sktalk/corpus/parsing/cha.py

```diff
@@ -5,32 +5,67 @@
 
 
 class ChaFile(InputFile):
+    TIMING_REGEX = r"\D(?P<timing>\d{1,9}_\d{1,9})\D"
+    # participant may not be repeated on each line
+    PARTICIPANT_REGEX = r"(^\*(?P<participant>[^:*]+)\:){0,1}"
```

Comment:
The `{0,1}` quantifier is equivalent to `?`.

```suggestion
    PARTICIPANT_REGEX = r"(^\*(?P<participant>[^:*]+)\:)?"
```